doc/tutorial/first_steps.md

```diff
@@ -0,0 +1,290 @@
+
+You can list all images that are available on this server with:
+
+    lxc image list ubuntu:
+
+See {ref}`images` for more information about the images that LXD uses.
+
+Now, let's start by launching a few instances.
+With *instance*, we mean either a container or a virtual machine.
+See {ref}`containers-and-vms` for information about the difference between the two instance types.
+
+For managing instances, we use the LXD command line client `lxc`.
+See [About `lxd` and `lxc`](lxd-lxc) if you are confused about when to use the `lxc` command and when to use the `lxd` command.
+
+1. Launch a container called "first" using the Ubuntu 22.04 image:
+
+       lxc launch ubuntu:22.04 first
+
+   ```{note}
+   Launching this container takes a few seconds, because the image must be downloaded and unpacked first.
+   ```
+
+1. Launch a container called "second" using the same image:
+
+       lxc launch ubuntu:22.04 second
+
+   ```{note}
+   Launching this container is quicker than launching the first, because the image is already available.
+   ```
+
+1. Copy the first container into a container called "third":
+
+       lxc copy first third
+
+1. Launch a VM using the Ubuntu 22.04 image:
+
+       lxc launch ubuntu:22.04 vm --vm
+
+   ```{note}
+   Even though you are using the same image name to launch the instance, LXD downloads a slightly different image that is compatible with VMs.
+   ```
+
+1. Check the list of instances that you launched:
+
+       lxc list
+
+   You will see that all but the third container are running.
+   This is because you created the third container by copying the first, but you didn't start it.
+
+   You can start the third container with:
+
+       lxc start third
+
+1. Query more information about each instance with:
+
+       lxc info first
+       lxc info second
+       lxc info third
+       lxc info vm
+
+1. We don't need all of these instances for the remainder of the tutorial, so let's clean some of them up:
+
+   1. Stop the second container:
+
+          lxc stop second
+
+   1. Delete the second container:
+
+          lxc delete second
+
+   1. Delete the third container:
+
+          lxc delete third
+
+      Since this container is running, you get an error message that you must stop it first.
+      Alternatively, you can force-delete it:
+
+          lxc delete third --force
+
+   1. Stop and delete the VM:
+
+          lxc stop vm
+          lxc delete vm
+
+See {ref}`instances-create` and {ref}`instances-manage` for more information.
+
+## Configure instances
+
+There are several limits and configuration options that you can set for your instances.
+See {ref}`instance-options` for an overview.
+
+Let's create another container with some resource limits:
+
+1. Launch a container and limit it to one vCPU and 192 MiB of RAM:
+
+       lxc launch ubuntu:22.04 limited -c limits.cpu=1 -c limits.memory=192MiB
```

Comment:
Would it be valuable to also have an example here how to change the root disk size?

Reply:
The quoted example is a container which doesn't have any rootfs quota assigned by default, I think. However for VMs, I always struggle to remember the syntax so copying the example from `doc/howto/instances_create.md` would be nice IMHO as that's a pretty common ask, I think.

Reply:
Agreed, for VMs. As @simondeziel to use the disk space quota feature one has to start considering the pros and cons of each storage driver. And on this demo we are using `lxd init --minimal` which will use a `dir` pool which doesn't support container disk quotas.

Reply:
Thank you.